package.json

```diff
@@ -45,7 +49,7 @@
     "gulp-connect": "^2.2.0",
     "gulp-ext-replace": "^0.1.0",
     "gulp-gh-pages": "^0.4.0",
-    "gulp-sass": "^1.2.0",
+    "gulp-sass": "^2.0.4",
```

Comment:
I think this should be part of this PR as it relates to fixing a problem with the build process and not really to updating to React 0.14